lib/barnes/railtie.rb

```diff
@@ -1,16 +1,30 @@
 require 'rails/railtie'
 require 'barnes/reporter'
+require 'barnes/resource_usage'
 
 module Barnes
   class Railtie < ::Rails::Railtie
-    config.barnes = Barnes::Reporter.new
+    config.barnes = {
+      :interval => 10,
+      :statsd => nil,
+      :panels => [],
+    }
 
     initializer 'barnes' do |app|
       require 'statsd'
 
-      app.config.barnes.logger ||= Rails.logger
-      Periodic.new config.barnes
-    end
+      sample_rate = config.barnes[:interval] / 60.0
```

Comment:
Why 60.0?

Reply:
It should probably not be hardcoded, I agree. We're using a 60s aggregation period. This should certainly be configurable with this as default.